e2e/exit-exam-modal.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { test, expect, type Page } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+
+test.use({ storageState: 'playwright/.auth/certified-user.json' });
+
+let page: Page;
```

Comment:
```suggestion
import { test, expect } from '@playwright/test';
import translations from '../client/i18n/locales/english/translations.json';

test.use({ storageState: 'playwright/.auth/certified-user.json' });
```

We also don't need a global `page` variable because we're already getting `page` from the test function args.

Reply:
@naomi-lgbt thanks  resolved